roles/openshift-applier/tasks/pre-check.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Exit if ansible version doesn't meet minimum requirements
+  fail:
+    msg: "openshift-applier requires at least Ansible 2.5 in order to proceed"
+  when:
+  - "ansible_version.full is version('2.5','<')"
+
+
+- name:  "Determine oc version"
+  shell: "oc version | sed -ne 's/^oc v*\(.*\)+.*$/\1/p'" 
+  register: oc_vers_check
+
+- name: "Check oc version for null output"
+  set_fact:
+    oc_ignore_unknown_parameters: false
+  when:
+  - oc_vers_check.stdout == ""
```

Comment:
Even though somewhat "ugly", we have done these checks as combined checks before and are in general better than doing 2 tasks - i.e.: for this it would be something like (note the beginning `-`)
```
- oc_vers_check is not defined or
  oc_vers_check.stdout is not defined or
  oc_vers_check.stdout|trim == "" or
  oc_vers_check.stdout is version('3.6','<=')

Reply:
Note I just updated the above to be all `and` 's ... as I believe if we we don't get a valid output, we should *not* set this var to `false`

Reply:
@oybed I updated these checks. I had been trying to do this as a bunch of `or` checks, which I think is what was causing my issues. My assumption is that if we couldn't get any valid outputs we should default to false (which sounds like isn't the case).